rotkehlchen/tests/exchanges/test_kraken.py

```diff
@@ -407,3 +413,15 @@ def mock_post(url, data):  # pylint: disable=unused-argument
         result, msg = kraken.query_balances()
         assert msg == ''
         assert result == {}
+
+
+def test_timestamp_deserialization():
+    """Test the function that allows to deserialize timestamp from different types"""
+    assert deserialize_timestamp_from_kraken("1458994442.2353") == 1458994442
```

Comment:
nitpick: Won't fix here since all tests green. Please use single quotes for string literals. (We have to use black at some point soon)